### ISSUE TITLE
Landcover related landuse should all show at zoom 9

### ISSUE THREAD

**Issue by nvkelso** [tilezen/vector-datasource#1813]
http://localhost:8007/#9/38.1648/-121.2671

We're showing some landcover type landuse starting at zoom 9 (like grassland), and others at zoom 10 and other zooms. They should all come in at the same zoom (9), when we switch to OSM data source.

![image](https://user-images.githubusercontent.com/853051/52156550-9a97a200-263d-11e9-8d3f-d23b2f428367.png)

Versus zoom 10, which adds farmland:

![image](https://user-images.githubusercontent.com/853051/52156561-c2870580-263d-11e9-92b0-86a7f6db82e2.png)

**Comment by nvkelso** [comment]
tl;dr

- **Looks like `farmland` is the big offender here. It needs to start at zoom 9 not 10.**
- We may also want to move `orchard` to zoom 9 as well.
- Generally we include the rest at zoom 9 already if they are large, and throttle the smaller ones for later zooms (eg `filter: { landuse: forest }` with `min_zoom: { max: [ 9, *tier2_min_zoom ] }`. That seems to be working for us, but it could also be creating holes for us.

**Research:**

Related to https://github.com/tilezen/vector-datasource/issues/1259 and https://github.com/tilezen/vector-datasource/pull/1788.

_From that issue's description of low zoom landcover:_

```
landuse IN
  farmland (and farm), modify the min to be 9 instead of 10.
  orchard, starts at 10 now, should be 9?
  forest – do we need a custom `tier2_min_zoom` to show all at zoom 9?
  residential - okay to start at 10, because of NE overlap
  commercial - okay to start at 10, because of NE overlap
  retail - okay to start at 10, because of NE overlap
  industrial - okay to start at 10, because of NE overlap
  meadow, starts at 9, but throttled
  vineyard, starts at 9, but throttled
natural IN
  wood – do we need a custom `tier2_min_zoom` to show all at zoom 9?
  sand - starts at 9, but throttled
  scree - starts at 9, but throttled
  shingle - starts at 9, but throttled
  bare_rock - starts at 9, but throttled
  heath - starts at 9, but throttled
  grassland - starts at 9, but throttled
  scrub - starts at 9, but throttled
natural IN
  wetland - starts at 9, but throttled
  mud - starts at 9, but throttled
```

**Comment by nvkelso** [comment]
I also wonder if we should be including all the "urban" content in a big union separate from any remapping – that way we'd get maximum `urban_area` polygon at mid-zooms, while preserving when individual overlays like hospital, school, parking lot come in?

See 12/37.9778/-122.0918.

![image](https://user-images.githubusercontent.com/853051/52157181-f401d000-2641-11e9-9b13-61bfe34a976f.png)

**Comment by nvkelso** [comment]
Most this pink `forest` is only visible at zoom 10 not zoom 9 around 10/32.8367/35.6391 in Israel.

![image](https://user-images.githubusercontent.com/853051/52157322-03354d80-2643-11e9-89ba-0cbc26d6e172.png)

**Comment by nvkelso** [comment]
Around Watsonville, California (12/36.8614/-121.7199)

![image](https://user-images.githubusercontent.com/853051/52189477-acb84280-27ed-11e9-9fc2-6363fc4be792.png)

**Comment by zerebubuth** [comment]
Related: #1794.

**Comment by nvkelso** [comment]
The final QA changes for min_zoom are verified. Here for farmland:

![image](https://user-images.githubusercontent.com/853051/52754495-98bfce00-2faf-11e9-92a1-397e8dcebb23.png)
